packages/ui/src/walkthrough-card/WalkthroughCard.css

```diff
@@ -0,0 +1,46 @@
+.ui-WalkthroughCard--CardContainer {
+  display: flex;
+  border-radius: var(--layout-border-radius-xlarge);
+  box-shadow: var(--surface-shadow-none), var(--surface-shadow-none);
```

Comment:
I'm seeing `box-shadow: var(--surface-shadow-medium);` in the designs. Also the definitions in `variables.css` already have both parts with a comma.

![Screenshot 2024-06-17 at 11 15 07 AM](https://github.com/bcgov/House-Innovations-HOUSNAV/assets/41478918/5f0229d0-d938-470a-9d71-2c3e83714645)

Reply:
Yep, that's what it transitions to on hover

Reply:
I'm seeing a transition from medium -> large on hover. Are you seeing something different?